src/explore-education-statistics-common/src/modules/release/components/NationalStatisticsSection.tsx

```diff
@@ -9,20 +9,25 @@ export default function NationalStatisticsSection({
     <>
       {showHeading && <h3>National statistics</h3>}
       <p>
-        The{' '}
-        <a href="https://www.statisticsauthority.gov.uk/">
-          United Kingdom Statistics Authority
-        </a>{' '}
-        designated these statistics as National Statistics in accordance with
+        These accredited official statistics have been independently reviewed by
         the{' '}
+        <a href="https://osr.statisticsauthority.gov.uk/what-we-do/">
```

Comment:
do we want this to be a React `<Link>` component?

Reply:
We can't easily use `<Link>` in common components because the admin and public apps have differnet link components as the public app uses NextJS but the admin is just React.

Reply:
ah ok